tests/boot/boot_from_pxe.pm

```diff
@@ -172,4 +172,19 @@ sub run {
     }
 }
 
+sub post_fail_hook {
+    my $self = shift;
+
+    if (check_var('BACKEND', 'ipmi') && check_var('VIDEOMODE', 'text')) {
+        select_console 'log-console';
+        save_screenshot;
+        script_run "save_y2logs /tmp/y2logs_clone.tar.bz2";
+        upload_logs "/tmp/y2logs_clone.tar.bz2";
+        save_screenshot;
```

Comment:
Isn't that the same that the parent post fail hook should already try?

Reply:
well, i really hope so, then i do not need this PR, but it does not. You can refer to its base class opensusebasetest.pm post_fail_hook for details.